src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,133 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private T[] arrayList;
+    int size = 0;
+    int emptyPlaces = 10;
+
+    public ArrayList() {
```

Comment:
The class should not begin with an empty line. Remove the empty line at the start of the class definition.